ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -633,18 +660,17 @@ public static Bytes32 get_randao_mix(BeaconState state, UnsignedLong epoch) {
    *     - Spec v0.4</a>
    */
   public static Bytes32 get_block_root(BeaconState state, UnsignedLong slot) {
+    checkArgument(
+        slot.compareTo(state.getSlot()) < 0,
+        "checkArgument threw an exception in get_block_root()");
     checkArgument(
         state
                 .getSlot()
                 .compareTo(slot.plus(UnsignedLong.valueOf(Constants.SLOTS_PER_HISTORICAL_ROOT)))
             <= 0,
         "checkArgument threw an exception in get_block_root()");
-    checkArgument(
-        slot.compareTo(state.getSlot()) < 0,
-        "checkArgument threw an exception in get_block_root()");
-    // Todo: Remove .intValue() as soon as our list wrapper supports unsigned longs
     return state
-        .getLatest_block_roots()
+        .getLatest_state_roots()
```

Comment:
This is wrong.

Reply:
ah dang - will fix. this was from the other PR before the change